src/test/three-components/ModelScene-spec.js

```diff
@@ -151,72 +118,26 @@ suite('ModelScene', () => {
     });
   });
 
-  suite('scaleModelToFitRoom', () => {
+  suite('alignModel', () => {
     test('does not throw if model has no volume', () => {
       scene.model.setObject(new Object3D());
-      scene.scaleModelToFitRoom();
+      scene.alignModel();
     });
 
     test('does not throw if model not loaded', () => {
       scene.model.modelContainer.add(dummyMesh);
       scene.model.updateBoundingBox();
-      scene.scaleModelToFitRoom();
-    });
-
-    test('increases the scale of a small object to fill the limit', () => {
```

Comment:
Holy test suite bloodbath, Batman 😱 

It's not necessarily bad to remove tests. Sometimes they will be obsolete and dead-weight. But, without additional context, I would say that these tests look at least conceptually relevant (even if their implementation might now be obsolete).

My recommendations here are:

 - It's usually better to evolve a test towards relevance rather than remove it
 - Our test suite is already sparse; a change with the scope of this one might even call for additional tests (esp. since issues are being closed out)
 - If you remove obsolete, dead-weight tests, explain why in the PR description for the benefit of your reviewers

Also added some guidelines here: https://github.com/GoogleWebComponents/model-viewer/wiki/Code-conventions#tests

Reply:
Fair enough. I removed these because really I combined them with another set of tests below. Once the logic was simplified, they ended up testing pretty much the same thing. More tests could always be added of course. If you see something that looks like it could use more, I’m happy to add it.

Reply:
A good place to start would be to add at least one test for each issue being closed out that will guard us against likely regressions.

For example, I might have expected to see a test added that asserts that the scale of the model _does not change_ as the element resizes. In fact, looking back this seems to be the spirit of the tests you removed (checking how the scale of the model changes).

Reply:
Sure thing, added.